configs/hoodi/lidov3-testnet.yaml

```
@@ -0,0 +1,1994 @@
+        UNSAFE_SET_EXITED_VALIDATORS_ROLE: *UNSAFE_SET_EXITED_VALIDATORS_ROLE
+        getAllNodeOperatorDigests: null
+        getAllStakingModuleDigests: []
+        getContractVersion: *MAX_UINT256
+        getDepositsAllocation: null
+        getLido: *ZERO_ADDRESS
+        getNodeOperatorDigests: null
+        getNodeOperatorSummary: null
+        getRoleAdmin: null
+        getRoleMember: null
+        getRoleMemberCount: null
+        getStakingFeeAggregateDistribution:
+          - 0
+          - 0
+          - 100000000000000000000
+        getStakingFeeAggregateDistributionE4Precision:
+          - 0
+          - 0
+        getStakingModule: null
+        getStakingModuleActiveValidatorsCount: null
+        getStakingModuleDigests: null
+        getStakingModuleIds: []
+        getStakingModuleIsActive: null
+        getStakingModuleIsDepositsPaused: null
+        getStakingModuleIsStopped: null
+        getStakingModuleLastDepositBlock: null
+        getStakingModuleMaxDepositsCount: null
+        getStakingModuleMaxDepositsPerBlock: null
+        getStakingModuleMinDepositBlockDistance: null
+        getStakingModuleNonce: null
+        getStakingModuleStatus: null
+        getStakingModuleSummary: null
+        getStakingModules: []
+        getStakingModulesCount: 0
+        getStakingRewardsDistribution: null
+        getTotalFeeE4Precision: 0
+        getWithdrawalCredentials: *ZERO_BYTES32
+        hasRole: null
+        hasStakingModule: null
+        supportsInterface: null
+
+    executionLayerRewardsVault:
+      name: LidoExecutionLayerRewardsVault
+      address: *executionLayerRewardsVault
+      checks:
+        LIDO: *lido
+        TREASURY: *aragonAgent
+
+    withdrawalQueueERC721:
+      name: WithdrawalQueueERC721
+      address: *withdrawalQueueERC721
+      proxyName: OssifiableProxy
+      implementation: *withdrawalQueueERC721ImplAddress
+      proxyChecks:
+        proxy__getAdmin: *aragonAgent
+        proxy__getImplementation: *withdrawalQueueERC721ImplAddress
+        proxy__getIsOssified: false
+      checks:
+        BUNKER_MODE_DISABLED_TIMESTAMP: *MAX_UINT256
+        DEFAULT_ADMIN_ROLE: *ZERO_BYTES32
+        FINALIZE_ROLE: *FINALIZE_ROLE
+        MANAGE_TOKEN_URI_ROLE: *MANAGE_TOKEN_URI_ROLE
+        MAX_BATCHES_LENGTH: 36
+        MAX_STETH_WITHDRAWAL_AMOUNT: 1000000000000000000000
+        MIN_STETH_WITHDRAWAL_AMOUNT: 100
+        ORACLE_ROLE: *ORACLE_ROLE
+        PAUSE_INFINITELY: *MAX_UINT256
+        PAUSE_ROLE: *PAUSE_ROLE
+        RESUME_ROLE: *RESUME_ROLE
+        STETH: *lido
+        WSTETH: *wstETH
+        balanceOf: null
+        bunkerModeSinceTimestamp: *MAX_UINT256
+        calculateFinalizationBatches: null
+        findCheckpointHints: null
+        getApproved: null
+        getBaseURI: ""
+        getClaimableEther: null
+        getContractVersion: 1
+        getLastCheckpointIndex: 2
+        getLastFinalizedRequestId: 2
+        getLastRequestId: 2
```

```suggestion
        getLastCheckpointIndex: null
        getLastFinalizedRequestId: null
        getLastRequestId: null
```

too fragile to check